json_schema_fuzz.py

```diff
@@ -17,6 +18,17 @@ def random_object(schema):
             object[key] = generate_json(value)
     return object
 
+def random_boolean(schema):
+    """Generate random JSON boolean."""
+    return random.choice([True, False])
+
+def random_string(schema):
+    """Generate random string."""
+    lowercase_letters = string.ascii_lowercase
+    word_length = random.randrange(1, (20+1))
```

Comment:
What is the purpose of `(20+1)`?

Reply:
I didn't mean to leave that in there! The snip of code was written in a different function originally where the user defined the highest length the string could be, I replaced the variable with 20 without thinking to delete the +1. The reason the +1 was there in the first place is that randrange() chooses from an exclusive range.